osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -209,6 +213,7 @@ public ScoreProcessor(Ruleset ruleset)
         {
             Ruleset = ruleset;
 
+            minimumResultCounts[HitResult.Miss] = MaxHits;
```

Comment:
I think you're meant to put this in `ScoreProcessor.Reset`

Reply:
Implemented in [ebede5c](https://github.com/ppy/osu/pull/28614/commits/ebede5c79d2ae137708f1bbce8be715d64804115)
Merge state